app/views/hub/clients/_client_header.html.erb

```diff
@@ -33,6 +33,9 @@
         </div>
       </div>
 
+      <% if @client.needs_itin_help_status == I18n.t("general.affirmative") %>
```

Comment:
People worry about [tri-state booleans](https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem). You have something even weirder: you have a four-state boolean! :D

In case it's useful, here are some ideas.

- You can return `true`, `false`, `nil`, or `:n_a` from `needs_itin_help_status`.
- You could return `:yes` `:no` `:missing` or `:n_a` from `needs_itin_help_status`.
- You could move `needs_itin_help_status` out of `Client` and into `HubClientPresenter`, and then you could examine the status and return the right I18n string.

Of these, I like that last option the most. Not a blocker.

Reply:
Update: Most of what I said above was nonsensical but I tuple'd with Catie.